src/org/opensolaris/opengrok/index/IndexDatabase.java

```diff
@@ -356,7 +349,12 @@ private void markProjectIndexed(Project project) throws IOException {
                 Message m = Message.createMessage("project");
                 m.addTag(project.getName());
                 m.setText("indexed");
-                m.write(env.getConfigHost(), env.getConfigPort());
+                try {
+                    m.write(env.getConfigHost(), env.getConfigPort());
+                } catch (ConnectException ce) {
```

Comment:
I'm not quite clear why this exception is now being swallowed in a Lucene upgrade.

Reply:
so this exception should be really a warning
since when implemented it forced a local webapp (or setup webapp) to be listening to update ... which really doesn't have to be a case, when you index from a different box and you don't want to update anyone, just create an index ... (e.g. parallel reindex from scratch)
I know that @vladak or @tulinkry might not like it, but I am just relaxing what they forced, since I don't think that an error of update should break or stop whole indexing ...

Reply:
OK I see. The PR looks good to me.